### PR TITLE
Switching To Infura's Currency API

### PR DESCRIPTION
**Pull request by 2-am-zzz**
Does not maintain parity yet, will need to write a migration or some other solution to navigate around this.

Thanks again to @strelok1 for the contributions, though we unfortunately do not have AUD on the new API (a request can be sent to the maintainers of api.infura.io).

### CODE REVIEW

**Comment by maurycyp** [comment]
@strelok1 has created an INFURA ticket for this task https://github.com/INFURA/infura/issues/34 , and we'll be tracking progress there.

**Comment by maurycyp** [comment]
Here's the ticket for ethkrw https://github.com/INFURA/infura/issues/35 .

**Comment by maurycyp** [comment]
As part of my work on https://github.com/INFURA/infura/issues/34 , we've added the following 5 new pairs:

ETHAUD - Australian dollar
ETHHKD - Hong Kong dollar
ETHSGD - Singapore dollar
ETHIDR - Indonesian rupiah
ETHPHP - Philippine peso

**Comment by tmashuang** [comment]
Getting a status 404 from https://api.infura.io/v1/ticker/ethUSD

File: app/scripts/controllers/currency.js [hunk 1]
```diff
@@ -45,10 +45,10 @@ class CurrencyController {
 
   updateConversionRate () {
     const currentCurrency = this.getCurrentCurrency()
-    return fetch(`https://api.cryptonator.com/api/ticker/eth-${currentCurrency}`)
+    return fetch(`https://api.infura.io/v1/ticker/eth${currentCurrency}`)
```
Comment (tmashuang) [inline app/scripts/controllers/currency.js]: Should this be `eth$-{currentCurrency}`

**Comment by maurycyp** [comment]
It's case-sensitive, please use https://api.infura.io/v1/ticker/ethusd .

**Review by kumavis — APPROVED** [review]

**Comment by maurycyp** [comment]
Two new pairs added:

ETH-KRW - South Korean won
ETH-CNY - Renminbi

https://api.infura.io/v1/ticker/ethkrw
https://api.infura.io/v1/ticker/ethcny

**Pull request merged**